packages/fscomponents/src/components/FilterList/FilterListDrilldown.tsx

```diff
@@ -18,18 +21,25 @@ const componentTranslationKeys = translationKeys.flagship.filterListDefaults;
 
 const defaultSingleFilterIds = [`cgid`];
 
+const closeIcon = require('../../../assets/images/clear.png');
+
 export interface FilterListDrilldownProps {
   items: FilterItem[];
   onApply: (selectedItems: Record<string, string[]>, info?: { isButtonPress: boolean }) => void;
   onReset: (info?: { isButtonPress: boolean }) => void;
+  onClose?: () => void;
   selectedItems?: Record<string, string[]>;
   style?: StyleProp<ViewStyle>;
+  closeButtonStyle?: StyleProp<ViewStyle>;
   resetButtonStyle?: StyleProp<ViewStyle>;
   applyButtonStyle?: StyleProp<ViewStyle>;
+  closeButtonImageStyle?: StyleProp<ImageStyle>;
   resetButtonTextStyle?: StyleProp<TextStyle>;
   applyButtonTextStyle?: StyleProp<TextStyle>;
+  closeIcon?: ImageURISource;
```

Comment:
Can this be `ImageSourcePropType` so we're not explicitly requiring a URI source?